.changeset/introduce_includes.md

```diff
@@ -1,44 +1,66 @@
 ---
-"@biomejs/biome": minor
+"@biomejs/biome": major
 ---
 
-Introduce `includes`.
+Replace `include` and `ignore` with `includes`.
 
-Biome allows users to `include` and `ignore` files in its configuration using glob patterns.
+The Biome configuration file allows users to specify which files should be processed using [glob patterns](https://en.wikipedia.org/wiki/Glob_(programming)).
+Prior to Biome 2.0, this was done using the `include` and `ignore` fields.
+In Biome 2.0, `include` and `ignore` are removed and replaced by `includes`.
+You can run `biome migrate` to convert `include` and `ignore` into `includes` automatically.
 
-For example, in the following configuration, all files of the `src/` directory are checked except the ones ending with the extension `.test.js`.
+`includes` uses a different glob pattern format that fixes [many](https://github.com/biomejs/biome/issues/2421) [issues](https://github.com/biomejs/biome/issues/3345) and limitations that Biome users reported.
+
+`includes` accepts an array of glob patterns.
+A glob pattern starting with a `!` is an exception.
+This replaces `ignore` patterns and allows users to create chains of include and ignore patterns.
+Thus, it is now possible to include again a file previously ignored.
+This was not possible with `include` and `ignore`, because `ignore` was always applied after `include`.
```

Comment:
```suggestion
This was not possible with `include` and `ignore`, because `ignore` was always applied before `include`.
```

Do you think the suggestion is correct?

Reply:
No, it is the reverse. We first include files and then we ignore some of them. Another way to see things is `ignore` takes priority over `include`.

Reply:
Yeah, that's what I knew it was, but the way it's phrased makes think otherwise. Can't you phrase it from the user's point of view instead of from the developer's point of view?